assets/sass/components/key-metrics/_googlesitekit-km-widget-tile.scss

```diff
@@ -219,3 +200,25 @@
 		}
 	}
 }
+
+.googlesitekit-tooltip.MuiTooltip-tooltip.googlesitekit-km-widget-tile-title__tooltip {
+	border-radius: $br-xs;
+	font-family: $f-primary;
+	font-size: $fs-body-sm;
+	font-weight: $fw-normal;
+	letter-spacing: $ls-xs;
+	line-height: $lh-body-sm;
+	max-width: 160px;
+	padding: 12px;
+
+	// Incorporating these styles within the preceding selector affects the button's appearance.
+	// Hence, the stylelint rule for no-descending-specificity is disabled for the following line.
+	/* stylelint-disable-next-line no-descending-specificity */
```

Comment:
I think the rule means that the selector you pulled out should go above the previous preceding selector rather than after – then the rule shouldn't need to be disabled.

Reply:
Good catch. Thanks 👍